lib/ui/painting.dart

```diff
@@ -2331,15 +2317,20 @@ class PathMetric {
 
 class _PathMeasure extends NativeFieldWrapperClass2 {
   _PathMeasure(Path path, bool forceClosed) {
-    currentContourIndex = -1; // PathMetricIterator will increment this the first time.
+    currentContourIndex = -1; // nextContour will incremenet this to the zero based index.
```

Comment:
typo: "increment"

Reply:
Argh.  I thought I pushed up the fix for this before merging.  Will fix...